### PR TITLE
All segmentation models supported

### PR DESCRIPTION
**Pull request by will-jl944**
1. Add pretrained weights for DeepLabV3P series, remove Xception from backbone choices temporarily since no pretrained weights is provided.
2. Keep up with PaddleSeg release/2.0
3. Add mixed_loss for segmentation models (CrossEntropy + LovaszSoftmax)
4. Add auxiliary loss for FastSCNN

### CODE REVIEW

File: tutorials/train/semantic_segmentation/deeplabv3p.py [hunk 1]
```diff
@@ -34,9 +34,7 @@
     transforms=eval_transforms,
     shuffle=False)
 
-deeplabv3p_backbone_list = [
-    'ResNet50_vd', 'ResNet101_vd', 'Xception65_deeplab'
-]
+deeplabv3p_backbone_list = ['ResNet50_vd', 'ResNet101_vd']
```
Comment (FlyingQianMM) [inline tutorials/train/semantic_segmentation/deeplabv3p.py]: backbone这里可以直接写'ResNet50_vd'，其余选项用户可以查看文档

File: paddlex/cv/models/segmenter.py [hunk 2]
```diff
@@ -323,4 +363,69 @@ def __init__(self,
...
+
+        params = {'backbone': backbone, 'align_corners': align_corners}
+        super(HRNet, self).__init__(
+            model_name='FCN', num_classes=num_classes, losses=losses, **params)
+        self.model_name = 'HRNet'
+
+
+class BiSeNetV2(BaseSegmenter):
+    def __init__(self, num_classes=2, align_corners=False):
+        losses = {
```
Comment (FlyingQianMM) [inline paddlex/cv/models/segmenter.py]: 这里也可以支持use_mixed_loss

File: paddlex/cv/models/segmenter.py [hunk 3]
```diff
@@ -292,29 +299,62 @@ def _postprocess(batch_pred, batch_origin_shape, transforms):
...
 
 class UNet(BaseSegmenter):
-    def __init__(self, num_classes=2, use_deconv=False, align_corners=False):
+    def __init__(self,
+                 num_classes=2,
+                 use_mixed_loss=False,
+                 use_deconv=False,
+                 align_corners=False):
+        if use_mixed_loss:
+            losses = [
```
Comment (FlyingQianMM) [inline paddlex/cv/models/segmenter.py]: use_mixed_loss可选范围：True，False，list of (tuple)
1. True，默认使用CrossEntropyLoss和LovaszSoftmaxLoss组合，且比例为0.8， 0.2
2. False, 默认使用CrossEntropyLoss
3. 如果为list, 则使用1种或多种loss组合，且比例可调配，e.g. [('CrossEntropyLoss', 0.8), ('LovaszSoftmaxLoss', 0.2)]
4. 判断支持的loss类型

File: paddlex/cv/models/segmenter.py [hunk 4]
```diff
@@ -323,4 +363,69 @@ def __init__(self,
...
-            model_name='DeepLabV3P', num_classes=num_classes, **params)
+            model_name='DeepLabV3P',
+            num_classes=num_classes,
+            losses=losses,
+            **params)
+
+
+class FastSCNN(BaseSegmenter):
+    def __init__(self, num_classes=2, align_corners=False):
+        params = {'align_corners': align_corners}
```
Comment (FlyingQianMM) [inline paddlex/cv/models/segmenter.py:391]: FastSCNN也可以支持mixed loss，需要check下在开放mixed loss时enable_auxiliary_loss是否需要开放。

File: tutorials/train/semantic_segmentation/deeplabv3p.py (hunk 1 above)
Comment (will-jl944) [reply tutorials/train/semantic_segmentation/deeplabv3p.py]: Done

File: paddlex/cv/models/segmenter.py (hunk 3 above)
Comment (will-jl944) [reply paddlex/cv/models/segmenter.py]: Done

File: paddlex/cv/models/segmenter.py (hunk 4 above)
Comment (will-jl944) [reply paddlex/cv/models/segmenter.py:391]: Done

File: paddlex/cv/models/segmenter.py (hunk 2 above)
Comment (will-jl944) [reply paddlex/cv/models/segmenter.py]: Done
